payments/credit-card/redirect.md

```diff
@@ -151,55 +154,55 @@ Content-Type: application/json
-|          | creditCard.noCvc                                     | boolean     | `true` if the CVC field should be disabled for this payment in the case a stored card is used; otherwise `false` per default. To use this feature it has to be enabled on the contract with Swedbank Pay.                                                                                                                                                                                                                                                                                                                                                                                                         |
+| Required | Property                                  | Type        | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| :------: | :---------------------------------------- | :---------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|  ✔︎︎︎︎︎  | operation                                 | string      | Purchase                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
All code should be fenced. We also need a description of each property – `Purchase` isn't enough.

```suggestion
|  ✔︎︎︎︎︎  | `operation`                               | `string`    | `Purchase`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```